src/Disks/ObjectStorages/MetadataStorageFromPlainRewritableObjectStorage.cpp

```diff
@@ -114,7 +114,7 @@ void loadDirectoryTree(
         getLogger("MetadataStorageFromPlainObjectStorage"),
         "Loaded directory tree for {} directories, found {} files",
         map.size(),
-        num_files);
+        num_files.load());
```

Comment:
Just curious, why are you adding load to all atomics?

Reply:
Because in the new version, with the minimal includes, fmt doesn't know how to print atomics. And I'd rather keep includes thin